XIPs/xip-reply-content-type.md

```diff
@@ -0,0 +1,60 @@
+---
+xip: 
+title: Reply Content Type
+description: Content type for reply messages
+author: Kunal Mondal
+status: Draft 
+type: Standards Track
+category: XRC
+created: 2023-03-24
+---
+
+## Abstract
+
+This XRC proposes a new content type for messages that enriches the messaging experience. It attempts to include only the bare minimum for clients to determine how to display such messages in order to provide flexibility for more rich types in the future.
```

Comment:
Abstract should summarize what is being proposed (https://github.com/xmtp/XIPs/blob/main/XIPs/xip-0-purpose-process.md#what-belongs-in-a-successful-xip). I don't think this paragraph does that. I'd expect to see something like `This XRC proposes a reply content type that can be used to ...`. Most of what's in this paragraph can be deleted without noticeable loss of information.

Reply:
Yeah, we have changed it. 
https://github.com/xmtp/XIPs/pull/22/commits/951c0fe63dc77faf09ee0d4e47acee3eb6345d7b